### PR TITLE
expose type, plugin_name, dependencies, version as properties

### PR DESCRIPTION
**Pull request by benstoltz**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -9,6 +9,17 @@ const zlib = require('zlib')
 const parseString = require('xml2js').parseString
 const FileSystem = require('../')
 
+test('type and plugin_name are accessible', t => {
+  const s3fs = new FileSystem()
```
Comment (dmfenton) [inline test/index.js:13]: this is not right. We need to be able to access `Filesystem.type` instead of `s3fs.type` We already have `s3fs.type`

**Comment by dmfenton** [comment]
Need to start a changelog with this PR

**Comment by benstoltz** [comment]
closes #10

File: CHANGELOG.MD [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.0.2] - 2016-04-13
```
Comment (dmfenton) [inline CHANGELOG.MD:5]: Need to add a link so this comparison will show up

Take a look at the raw markdown in one of the other projects

**Comment by dmfenton** [comment]
👍

**Pull request merged**